### PR TITLE
Add detailed message for ImportError in __init__

### LINKED ISSUE #46: Detailed message on ImportError
I think it should be better rather than just showing
https://github.com/ManimCommunity/ManimPango/blob/f5dcf9472215eb2cb5f6ccfe5f56be47fbc30f3c/manimpango/__init__.py#L12-L17
I think it should be better to give some information like the python version and manimpango's version. Also, that should ask people to make an issue here or contact us on Discord.

### PR DESCRIPTION
**Pull request by RickyC0626**
Closes #46

### CODE REVIEW

File: manimpango/__init__.py [hunk 1]
```diff
@@ -14,4 +15,9 @@
     from .enums import *  # noqa: F403,F401
     from .register_font import *  # noqa: F403,F401
 except ImportError:  # pragma: no cover
-    raise ImportError("Couldn't load the necessary Shared Libraries.")
+    py_ver = ".".join(map(str, sys.version_info[:3]))
+    error = "Couldn't load the necessary shared libraries.\n"
+    error += f"ManimPango v{__version__}, Python v{py_ver}\n\n"
+    error += "Please contact us at https://discord.gg/mMRrZQW or "
+    error += "create an issue at https://github.com/ManimCommunity/ManimPango/issues\n"
```
Comment (naveen521kk) [inline manimpango/__init__.py:22]: How about using `"""` multi-line comment rather than doing so?

File: manimpango/__init__.py [hunk 2]
```diff
@@ -14,4 +15,9 @@
     from .enums import *  # noqa: F403,F401
     from .register_font import *  # noqa: F403,F401
 except ImportError:  # pragma: no cover
```
Comment (naveen521kk) [inline manimpango/__init__.py:17]: we should also get the original error message and append to end of message

File: manimpango/__init__.py (hunk 1 above)
Comment (RickyC0626) [reply manimpango/__init__.py:22]: Good idea

File: manimpango/__init__.py (hunk 2 above)
Comment (naveen521kk) [reply manimpango/__init__.py:17]: Have a look at how numpy does: https://github.com/numpy/numpy/blob/cb557b79fa0ce467c881830f8e8e042c484ccfaa/numpy/core/__init__.py#L27

Comment (RickyC0626) [reply manimpango/__init__.py:17]: > Have a look at how numpy does: https://github.com/numpy/numpy/blob/cb557b79fa0ce467c881830f8e8e042c484ccfaa/numpy/core/__init__.py#L27

Do you want it to be as verbose as numpy or just have similar elements?

Comment (naveen521kk) [reply manimpango/__init__.py:17]: I would like it to be verbose as numpy.

**Review by naveen521kk — APPROVED** [review]

**Pull request merged**